docs/en/Alluxio-Storage.md

```diff
@@ -0,0 +1,229 @@
+title: Alluxio Storage
+nickname: Alluxio Storage
+group: Features
+priority: 1
+---
+
+* Table of Contents
+{:toc}
+
+Alluxio manages the local storage, including memory, of Alluxio workers to act as a distributed
+buffer cache. This fast data layer between user applications and the various under storages results
+in vastly improved I/O performance.
+
+The amount and type of storage for each Alluxio node to manage is determined by user configuration.
+Alluxio also supports tiered storage, which makes the system storage media aware, enabling data
+storage optimizations similar to L1/L2 cpu caches.
+
+## Configuring Alluxio Storage
+
+The easiest way to configure Alluxio storage is to use the default single-tier mode.
+
+Out-of-the-box Alluxio will provision a ramdisk on every worker and take a percentage of the
+system's total memory. This ramdisk will be used as the only storage medium allocated to each
+Alluxio worker.
+
+Alluxio storage is configured through Alluxio's configuration in `alluxio-site.properties`. See the
+[configuration docs](Configuration-Settings.html) for detailed information.
+
+A common modification to the default is to explicitly set the ramdisk size. For example, to set the
+ramdisk size to be 16GB on each worker:
+
+```
+alluxio.worker.memory.size=16GB
+```
+
+Another common change is to specify multiple storage media, such as ramdisk and SSDs. We will need
+to update `alluxio.worker.tieredstore.level0.dirs.path` to take specify each storage medium we want
+to use as a storage directory. For example, to use the ramdisk (mounted at `/mnt/ramdisk`) and 2
+SSDs (mounted at `/mnt/ssd1` and `/mnt/ssd2`):
+
+```
+alluxio.worker.tieredstore.level0.dirs.path=/mnt/ramdisk,/mnt/ssd1,/mnt/ssd2
+```
+
+After updating the storage media, we need to indicate how much storage is allocated for each storage
+directory. For example, if we wanted to use 16 GB on the ramdisk and 100 GB on each SSD:
+
+```
+alluxio.worker.tieredstore.level0.dirs.quota=16GB,100GB,100GB
+```
+
+Note that the ordering of the quotas must match with the ordering of the paths.
+
+## Eviction
+
+Because Alluxio storage is designed to be volatile, there must be a mechanism to make space for new
+data when Alluxio storage is full. This is termed eviction.
+
+There are two modes of eviction in Alluxio, asynchronous (default) and synchronous. You can switch
+between the two by enabling and disabling the space reserver which handles asynchronous eviction.
+For example, to turn off asynchronous eviction:
+
+```
+alluxio.worker.tieredstore.reserver.enabled=false
+```
+
+Asynchronous eviction relies on a periodic space reserver thread in each worker to evict data. It
+waits until the worker storage utilization reaches a configurable high watermark. Then it evicts
+data based on the eviction policy until it reaches the configurable low watermark. For example, if
+we had the same 16+100+100=216GB storage configured, we can set eviction to kick in at around 200GB
```

Comment:
explain that `200GB` is derived by `216GB*0.9`, similar for the value `160GB`?

Reply:
Updated, thanks.